src/components/TodoContainer.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable react/no-this-in-sfc */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-use-before-define */
```

Comment:
- [x] Linters are tools that perform static analysis on software to detect and report adherence to a set of coding best practices. Instead of disabling eslint, it would be preferable to resolve the issue.